src/LineUI/LineUnit.tsx

```diff
@@ -146,6 +148,15 @@ const LineUnit : React.FC<ILineUnitProps> = (props) => {
     x: (x2 + x1) / 2,
     y: (y2 + y1) / 2
   };
+
+  const centerIconCordinates = () => {
```

Comment:
There's a typo on Coordinates (add an 'o'). You may want to rename this though when renaming the prop as commented elsewhere.